### DISCUSSION TITLE
ERROR 36

### DISCUSSION THREAD

**Discussion by iamFERU** [JunkFood02/Seal#1856] [General]
App version: 1.13.1 (11312)
Device information: Android 15 (API 35)
Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]
Yt-dlp version: 2024.11.09.232836

URL: https://x.com/HasnaZaruriHai/status/1855249419601342507?s=09
ERROR: Unable to download video: [Errno 36] File name too long: '/storage/emulated/0/Download/Seal/tmp/HasnaZarooriHai🇮🇳 - मैंने तो सुना था कि जया किशोरी जी कथा सुनाती हैं,  मगर ये क्या है？ 😧😧 [1855249276567228416].fhls-audio-128000-Audio.mp4.ytdl'

**Comment by bachdv** [comment]
--

Đỗ Văn Bách

*📱*0962.096.339

*Simple is the best!*



Vào CN, 10 thg 11, 2024 lúc 20:55 iamFERU ***@***.***> đã
viết:

**Reply by iamFERU** [reply]
Sorry, I didn't get you.
